ScnSoft-Education/LeetCodeTraining.Tests/MedianOfTwoArraysTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿using LeetCodeTraining.MedianOfTwoArrays;
+
+namespace LeetCodeTraining.Tests
+{
+    public class MedianOfTwoArraysSolutionTest

```

Comment:
Please rename the class to be Test at the and to consistency with other test classes.

Reply:
Got it. Now its renamed.